doc/overview/glossary.rst

```diff
@@ -59,9 +62,15 @@ Glossary
   Ring Buffer
     The ring buffer is a data structure that allows communication between the (unprotected) host and the enclave. Data that is written to one side can be read on the other. Only specific types of messages are supported to make sure each package that goes across is read by the right process in the right way.
 
+  RPC
```

Comment:
Should `REST` be listed in the glossary too?

Reply:
can do